src/lib/types/validator.ts

```diff
@@ -1,6 +1,9 @@
 import { z } from "zod";
 
-import { zValidatorAddr } from "./addrs";
+import { snakeToCamel } from "lib/utils/formatter/snakeToCamel";
```

Comment:
Let's import from `lib/utls`

Reply:
cannot import like that, getting cyclic warning